frontend/packages/kubevirt-plugin/src/hooks/use-render-vnc-console.ts

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { useEventListener } from './use-event-listener';
+
+interface RenderVNCConsoleArguments {
+  shouldBeFullScreen: boolean;
+  initValue?: boolean;
+}
+
+export const useRenderVNCConsole = ({
+  shouldBeFullScreen,
+  initValue,
+}: RenderVNCConsoleArguments): boolean => {
+  const value = !!localStorage.getItem('isFullScreenVNC') || initValue;
+  const [renderVNCConsole, setRenderVNCConsole] = React.useState<boolean>(value);
+
+  useEventListener(window, 'storage', () => {
+    setRenderVNCConsole(!!localStorage.getItem('isFullScreenVNC'));
```

Comment:
will it work with multiple consoles open from different VMs ?

Reply:
yes, in updated version, it is.